client/apps/edit/components/content/index.coffee

```diff
@@ -54,9 +58,15 @@ EditContent = React.createClass
       )
       SectionList {
         sections: @props.article.sections
-        saveArticle: @saveArticle
         article: @props.article
         channel: @props.channel
       }
+      React.createElement(
```

Comment:
Nitpick—It's _slightly_ odd to me to read the `createElement`s inside `render` here—it might be nicer to reassign these at the top of the module, e.g. `SectionFooter = React.createElement(SectionFooter)'. That said, ideally, we'd just migrate these to JSX anyways, so I won't hold anything up for it. Just food for thought.

Reply:
Yep this whole component is converted to es6 in the open redux PR, so these all will disappear shortly.